common/predictive-text/.gitignore

```diff
@@ -68,3 +68,4 @@ index.js
 test.js
 lmlayer.js
 *.js.map
+embedded_worker.js
```

Comment:
Any reason we're specifying particular *.js files, instead of setting up a more general pattern like the following?

```
**/*.js
!testing/**/*.js
!unit_tests/**/*.js
```

I've tested it locally, and this should accomplish what we want.

References:

https://stackoverflow.com/questions/29269705/git-ignore-files-with-particular-extension-in-a-particular-folder-name
https://stackoverflow.com/questions/3203228/git-ignore-exception

Reply:
No reason; I just haven't cleaned up the `.gitignore` yet. Incorporating your changes!